packages/@jsii/go-runtime/jsii-runtime-go/api/api.go

```diff
@@ -64,28 +64,3 @@ type EnumRef struct {
 type WireMap struct {
 	MapData map[string]interface{} `json:"$jsii.map"`
 }
-
-type Callback struct {
-	CallbackID *string        `json:"cbid"`
-	Cookie     *string        `json:"cookie"`
-	Invoke     InvokeCallback `json:"invoke"`
-	Get        GetCallback    `json:"get"`
-	Set        SetCallback    `json:"set"`
-}
-
-type InvokeCallback struct {
-	Method    string        `json:"method"`
-	Arguments []interface{} `json:"args"`
-	ObjRef    ObjectRef     `json:"objref"`
-}
-
-type GetCallback struct {
-	Property string    `json:"property"`
-	ObjRef   ObjectRef `json:"objref"`
-}
-
-type SetCallback struct {
-	Property string      `json:"property"`
-	Value    interface{} `json:"value"`
-	ObjRef   ObjectRef   `json:"objref"`
-}
```

Comment:
Those were all moved & hidden as implementation details of the kernel response unmarshaling.